### PR TITLE
Balboa spa document update for new temperature range control (Core PR #115285)

### PR DESCRIPTION
**Pull request by kotope**
## Proposed change

Update Balboa spa documentation for new temperature range control

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/115285
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

Link to previous PR that I closed due failure to resolve the target branch change: https://github.com/home-assistant/home-assistant.io/pull/32236

### CODE REVIEW

**Comment by kotope** [comment]
CI fails on ruby version. Is it just me (and my commit) or is there something wrong with the CI env?

File: source/_integrations/balboa.markdown [hunk 1]
```diff
@@ -29,6 +31,7 @@ There is currently support for the following device types within Home Assistant:
 - Climate
 - Fan (Pumps/Jets)
 - Light
+- Select (Low/High temperature range)
```
Comment (c0ffeeca7) [inline source/_integrations/balboa.markdown]:
```suggestion
- Select (Low/high temperature range)
```

**Comment by kotope** [comment]
The CI still fails on ruby version. Is there something I need to do @c0ffeeca7 ?

**Comment by c0ffeeca7** [comment]
> The CI still fails on ruby version. Is there something I need to do @c0ffeeca7 ?

No, nothing you can do I don't think. Just wait. I need help on this. But the earliest I could get someone to look into this is next week.

**Review by frenck — APPROVED** [review]
Thanks, @kotope 👍 

../Frenck

**Pull request merged**